src/unix/lwt_main.mli

```diff
@@ -28,6 +28,18 @@ val run : 'a Lwt.t -> 'a
       then returns the value returned by the thread. If [t] fails with
       an exception, this exception is raised.
 
+      Every program that uses Lwt should always use this function to launch
+      their threads, otherwise correct scheduling is not guaranteed.
```

Comment:
If I understand correctly, this is not true when targeting Javascript via js_of_ocaml.  It is true for native and bytecode compilation.

Reply:
Indeed, JS environments have the equivalent of `Lwt_main.run` built into them, conversely `Lwt_main.run` isn't even available when targeting JS.